libs/platform/src/lib/components/table/table.component.scss

```diff
@@ -274,13 +284,21 @@
         background-color: var(--sapList_HeaderBackground, #f2f2f2);
     }
 
+    .fdp-table__header,
+    .fdp-table__body,
+    .fdp-table__body-hs {
+        border: 0.0625rem solid #e4e4e4;
+        border: var(--sapList_BorderWidth, 0.0625rem) solid var(--sapList_BorderColor, #e4e4e4);
+
+        @include fd-rtl() {
+            border-left: 0.0625rem solid #e4e4e4;
+            border-left: var(--sapList_BorderWidth, 0.0625rem) solid var(--sapList_BorderColor, #e4e4e4);
```

Comment:
You are using the same value 4 times. May be you create scss variable for border value?